packages/core/src/coverage/index.ts

```diff
@@ -60,7 +60,8 @@ export interface SchemaCoverage {
     numTypesCoveredFully: number;
     numTypesCovered: number;
     numFields: number;
-    numFiledsCovered: number;
+    numFieldsCovered: number;
+    numFiledsCovered: number;  // deprecate and remove in next major version
```

Comment:
we can use JSDoc `@deprecated`, this way it will show up in IDEs

Reply:
@jirikuncar